script.check.theme/default.py

```diff
@@ -0,0 +1,11 @@
+## -*- coding: utf-8 -*-
+import xbmc
+import xbmcvfs
+import xbmcgui
+
+themepath =  xbmc.getInfoLabel( "listitem.path" ) + "theme" + ".mp3"
```

Comment:
would be nice if you could use the `os` module instead of concatenating strings:

```python
import os

themepath =  os.path.join(xbmc.getInfoLabel( "listitem.path" ), "theme" + ".mp3")
```